### ISSUE TITLE
Self-closing trust-dns issues

### ISSUE THREAD

**Issue by djc** [google/oss-fuzz#10857]
For trust-dns, we've recently seen a bunch of issues that are opened by the bot and then closed a few days later, without there being any (relevant) activity in the repo. Any clue why this could be? It seems pretty unexpected?

https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=61513
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=61350
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=61381

cc @bluejekyll @manunio

**Comment by djc** [comment]
No, they are definitely irrelevant. Also, those fixes predate the issue by quite a bit...

**Comment by oliverchang** [comment]
> No, they are definitely irrelevant. Also, those fixes predate the issue by quite a bit...

Is it possible that the dependency change in https://github.com/bluejekyll/trust-dns/compare/9e647103241a11082ffa7ff69489b8f1f683119b...3f9a68c15e76dc05739de22390eb58ec8ba43b16 caused the actual error surfaced to change? ClusterFuzz would consider that a different issue and potentially cause this closure.

Re the issue dates -- they actually were actually first seen a lot earlier (if you check the "Detailed report" links). We delayed in filing issues for them because there were other very similar open issues in the issue tracker already.

**Comment by djc** [comment]
The fuzz test only tests that parsing a DNS message works and round-trips correctly. The dependency change is for a dependency on the TLS protocol path which is not exercised by the fuzzer. The other commit is to the trust-dns-server crate which is not pulled in by the fuzzer (which only depends on trust-dns-proto).
